src/multigrid_iter.cpp

```diff
@@ -224,14 +224,14 @@ void gmgpolar::multigrid_iter()
         std::cout << "2-norm of error = " << nrm_2 << std::endl;
```

Comment:
gyro::icntl[Param::verbose]>0.
Or maybe Param::check_error should "overwrite" verbose =0 in this case. Then this is fine I think.